src/Components/Patient/PatientRegister.tsx

```diff
@@ -947,6 +983,38 @@ export const PatientRegister = (props: PatientRegisterProps) => {
             label={id !== 1 ? title : "NONE"}
           />
         </div>
+        {/* TB */}
+        {id === 12 && (field("medical_history").value ?? []).includes(id) && (
+          <div>
+            <SelectFormField
+              {...field("tb_status")}
+              position="above"
+              placeholder={"Status"}
+              options={["Active", "Old"]}
+              optionLabel={(o: any) => o}
+              optionValue={(o: any) => o}
```

Comment:
`any` type is not allowed especially when used along with newly built components. Newly built components has good type safety to prevent accidental errors.